src/SignalR/SignalR/SignalR.csproj

```
@@ -6,12 +6,22 @@
 
   <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
 
-  <PropertyGroup>
-    <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
-  </PropertyGroup>
+	<PropertyGroup>
+		<RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
+	</PropertyGroup>
```

Nit: please don't mix space and tab
```suggestion
  <PropertyGroup>
    <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
  </PropertyGroup>
```